sdk/search/search-documents/test/README.md

```diff
@@ -0,0 +1,42 @@
+# Testing
+
+To run our tests, first install the dependencies by running `npm install` (or `rush install`) at the root of the source code of this library.
+With the dependencies installed, you'll be able to run the unit tests and integration tests.
+
+## Unit tests
+
+To run unit tests, go to the root of this library and run `npm run unit-test`.
+This will run tests in a purely nodejs environment, and in a browser environment using [Karma](https://karma-runner.github.io/latest/index.html).
+These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses.
+You won't need to have any environment configuration, nor internet access, for these tests to run.
+
+Keep in mind that if you modify the test files, the API shape of this library, or the underlying network calls, the recordings need to be re-generated.
+You can read more in the section: [Regenerating recordings](#regenerating-recordings).
+
+## Integration tests
+
+Our integration tests will run against the live resources, which are determined by the environment variables you provide.
+
+To run the integration tests, you will need to execute `npm run integration-test`.
+
+To generate new resources for your tests, follow the procedure at [Integration Testing with live services](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#integration-testing-with-live-services).
+
+The environment variables needed are:
+
+- `SEARCH_API_ADMIN_KEY`: The primary key of your Azure Search account.
+- `SEARCH_API_ADMIN_KEY_ALT`: The secondary key of your Azure Search account.
```

Comment:
we don't technically use this at the moment, so maybe we can mark it as optional?

Reply:
For sure!

Reply:
same with TEXT_ANALYTICS_API_KEY_ALT

Reply:
Added `(optional)` to both. Let me know if that's ok!